tests/PolyType.SourceGenerator.UnitTests/NumberedLineWriter.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+﻿// Copyright (c) Microsoft Corporation. Full rights reserved.
```

Comment:
Seems this got subbed by accident
```suggestion
﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```